src/components/PersonCards.js

```diff
@@ -2,35 +2,51 @@ import React from 'react'
 import A from '../components/A'
 import Dropdown from '../components/Dropdown'
 import Masonry from 'react-masonry-component'
+import { humanDate } from '../util/text'
+import { tagUrl } from '../routes'
+import { isEmpty } from 'lodash'
 
-const PersonCards = ({ people, menus, subtitles }) => {
+const PersonCards = ({ people, menus, subtitles, slug }) => {
   return <div className='person-cards masonry-container'>
     <Masonry options={{transitionDuration: 0}}>
       {people.map(person => <div key={person.id} className='masonry-item-wrapper'>
         <PersonCard person={person}
           menu={menus && menus[person.id]}
-          subtitle={subtitles && subtitles[person.id]}/>
+          subtitle={subtitles && subtitles[person.id]}
+          slug={slug}
+          />
       </div>)}
     </Masonry>
   </div>
 }
 
 export default PersonCards
 
-export const PersonCard = ({ person, menu, subtitle }) => {
-  let { id, avatar_url, name, bio, isModerator } = person
+export const PersonCard = ({ person, menu, subtitle, slug }) => {
+  let { id, avatar_url, name, bio, isModerator, joined_at, createdTags, offerCount } = person
+  var role_and_age = isModerator ? 'Moderator' : 'Member'
```

Comment:
maybe just have this in the template below instead of storing it in a transient variable?